src/protocol_version.rs

```diff
@@ -1,10 +1,10 @@
-use std::sync::Mutex;
-
 use crc::crc32;
 use lazy_static;
 
 pub use net::constants::PROTOCOL_VERSION;
 
+use std::sync::Mutex;
```

Comment:
I believe this use should go above the pub use.

Reply:
According to how we, at least me and Fletcher, agreed on some format about how we should order our imports. And this is the way it should be as described in that document. I've sent it in discord.